packages/config/src/api/site_info.ts

```diff
@@ -29,14 +29,47 @@ type GetSiteInfoOpts = {
 export const getSiteInfo = async function ({
   api,
   siteId,
+  accountId,
   mode,
-  siteFeatureFlagPrefix,
   context,
   offline = false,
   testOpts = {},
+  featureFlags = {},
 }: GetSiteInfoOpts) {
   const { env: testEnv = false } = testOpts
 
+  const useV2Endpoint = !!accountId && featureFlags.cli_integration_installations_meta
```

Comment:
How would this work with buildbot? would we have the same feature flag?

Reply:
yes buildbot has access to the same feature flags